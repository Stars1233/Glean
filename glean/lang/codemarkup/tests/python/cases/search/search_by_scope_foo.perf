{
  "@generated": null,
  "facts_searched": {
    "python.DeclarationDefinition.4": 1,
    "python.DeclarationLocation.4": 1,
    "python.Name.4": 2,
    "python.SName.4": 2,
    "python.SearchFunctionByName.4": 1,
    "python.SearchModuleByName.4": 2
  },
  "full_scans": [ ]
}