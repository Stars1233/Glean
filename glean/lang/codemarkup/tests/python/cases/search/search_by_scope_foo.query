query: codemarkup.search.SearchByScope { name = "foo", scope = ["some","foo"], searchcase = Sensitive, language = Python }
perf: true
transform: [normord, []]
