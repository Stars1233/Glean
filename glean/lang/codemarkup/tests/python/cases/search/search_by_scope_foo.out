[
  "@generated",
  {
    "key": {
      "entity": {
        "python": {
          "decl": { "func": { "key": { "name": { "key": "some.foo.foo" } } } }
        }
      },
      "kind": 13,
      "language": 4,
      "location": {
        "file": { "key": "some/foo.py" },
        "location": { "span": { "length": 28, "start": 170 } },
        "name": "some.foo.foo"
      },
      "name": "foo",
      "scope": [ "foo", "some" ],
      "searchcase": 0
    }
  }
]